bn-expansions/src/EM/selective-discards.ts

```diff
@@ -3,34 +3,44 @@ import { flattenDeep, get, set } from 'lodash';
 import { groupingsToSpeciesDummyMap } from './helpers/maps';
 import { Catches, Disposition, sourceType } from '@boatnet/bn-models';
 
-const expansionCatchIds = ['5000']; //TODO maybe get this from some couch view or keep as a hardcoded value
+const dbConfig = require('../../dbConfig.json').dbConfig;
+const couchDB = require('nano')(dbConfig.login);
+const masterDev = couchDB.db.use('master-dev');
+
 const jp = require('jsonpath');
 
 export class selectiveDiscards implements BaseExpansion {
-    expand(params: ExpansionParameters): Catches {
+    async expand(params: ExpansionParameters): Promise<Catches> {
```

Comment:
does this imply the other expansions should be updated to return promises?

Reply:
removed async, moved database queries to tripsAPI so don't need async here.